sphinx-docs/installguide/install_linux.rst

```diff
@@ -7,13 +7,13 @@ Linux Installation Guide
 #. Check if Python is already installed with *python -V*.
 #. If not already installed or outdated, install Python v2.6 (*sudo apt-get install python2.6*) or v2.7 (*sudo apt-get install python2.7*).
 	* Or use your Distro's Package Manager by searching for *Python*.
-#. If not already installed or outdated, install Git with *apt-get install git-core*.
+#. If not already installed or outdated, install Git with *sudo apt-get install git-core*.
 	* Or use your Distro's Package Manager by searching for *Git*.
 #. Switch to the directory in which you wish to install KA-Lite.
 #. Enter *git clone https://github.com/learningequality/ka-lite.git* to download KA Lite.
-#. Switch into the newly downloaded ka-lite directory
+#. Switch into the newly downloaded ka-lite directory with *cd ka-lite*
 #. Run the install script with *./setup_unix.sh*.
-#. **IF** you want the server to start automatically in the background after your server starts then:
+#. **IF** you want the server to start automatically in the background  when your system boots and you did not choose this option during setup then:
```

Comment:
​I fixed the space.​ And you are very welcome, I am glad to be able to help
in a small way, thank you very much for all the work you are doing on this.


Website <http://togethertech.org/>  //  Facebook
<https://www.facebook.com/TogetherTechnologies>  //  Instagram
<http://instagram.com/togethertech>

On Thu, Feb 26, 2015 at 12:13 AM, Michael Gallaspy <notifications@github.com
> wrote:

> In sphinx-docs/installguide/install_linux.rst
> <https://github.com/MCGallaspy/ka-lite/pull/18#discussion_r25384831>:
>
> >  #. Run the install script with *./setup_unix.sh*.
> > -#. **IF** you want the server to start automatically in the background after your server starts then:
> > +#. **IF** you want the server to start automatically in the background  when your system boots and you did not choose this option during setup then:
>
> Hey @dewetb <https://github.com/dewetb>, these changes look fine to me,
> but there's an extra space here. Can you just remove that? Then if you push
> the changes to this same branch on your own fork it should be automatically
> reflected in this PR. I'll be happy to merge after that. Thanks again!
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/MCGallaspy/ka-lite/pull/18/files#r25384831>.
>